setup.py

```diff
@@ -39,7 +39,7 @@ def run(self):
     # SFTP
     "pysftp>=0.2.0,<0.3",
     # Utils
-    "pandas",
+    "pandas<1.4",
```

Comment:
can you please add a comment around why these versions are pinned? it's super helpful to know what things will break once we one to upgrade them.

Reply:
Ahhh yes 👍